src/components/backgrounds/BackgroundPreviewSettings.tsx

```diff
@@ -144,6 +150,11 @@ const BackgroundPreviewSettings = ({
   };
 
   const onChange = (newValue: string) => {
+    dispatch(
+      settingsActions.editSettings({
+        previewAsMono: (newValue == "0"),
+      })
+    );
     dispatch(editorActions.setPreviewAsSceneId(newValue));
   };
```

Comment:
I'm also using the "preview as" drop down selection to drive the `previewAsMono` editor setting so if you select it here or in the sprites page, the scenes will be in monochrome when you return to the world page.